pkg/api/object_example_test.go

```diff
@@ -42,13 +42,26 @@ func (o *ExampleObject) EndpointURL(ctx context.Context) (*url.URL, error) {
 	return url.Parse("/example/v1")
 }
 
+// GetIdentifier returns the identifier of an object
+// The Makefiles `generate` target creates this method for objects that have a anxcloud:"identifier" tagged field
+// Objects with multiple identifying fields can have this method overwritten to return an operation dependent identifier
+// The provided context might not always have the operation set. In this case the method must return the primary identifier
```

Comment:
Those sentences all lack a `.` at the end - which is fine for me when it's the only sentence in a comment, but not in this case.

Reply:
... did I really comment that twice? oh :deer: